### DISCUSSION TITLE
Adding target for Discord Bot integration in HA

### DISCUSSION THREAD

**Discussion by mstewart14** [mundschenk-at/ha-valetudo-blueprints#17] [General]
Discord integration requires a "target" under data to get the message to a particular channel.  The input is a string (a long integer channel ID)
I can figure out how to add the input to the blueprint but getting the target under data for sending the message is not working for me.  
Can you help me figure this out or can you add it?  
I'm happy to contribute if I can.  
Thank you for the blueprint, this works perfectly with the app.

**Comment by mundschenk-at** [comment]
Please create a [Notification group](https://www.home-assistant.io/integrations/group/#notify-groups) in YAML, the example already included the `target` data.

**Comment by mstewart14** [comment]
sigh... thank you, finally figured out the syntax to get that working.  feel free to make this a discussion or get rid of it.  
for those interested the notification group looks like:
```
# notify groups
notify:
  - name: my_discord_notification_group
    platform: group
    services:
      - service: my_discord_service
        data:
          target: "1234567890123456789" #discord channel ID
